src/test/java/io/rsocket/reactor/aeron/RsocketClientRunner.java

```diff
@@ -29,20 +31,35 @@ public static void main(String[] args) throws Exception {
               rSocket -> {
                 System.err.println("start " + rSocket);
 
-                rSocket
-                    .requestStream(DefaultPayload.create("Hello"))
-                    .log("receive ")
-                    .map(Payload::getDataUtf8)
-                    .doOnNext(System.out::println)
-                    .take(10)
-                    .then()
-                    .doFinally(signalType -> rSocket.dispose())
-                    .then()
+                //                rSocket
+                //                    .requestStream(DefaultPayload.create("Hello"))
+                //                    .log("receive ")
+                //                    .map(Payload::getDataUtf8)
+                //                    .doOnNext(System.out::println)
+                //                    .take(10)
+                //                    .then()
+                //                    .doFinally(signalType -> rSocket.dispose())
+                //                    .then()
+                //                    .subscribe();
+
```

Comment:
Instead of commenting out `requestStream` just create similar runner (maybe `RsocketRequestStreamClientRunner`), and this runner rename to `RsocketRequestResponseClientRunner` .

Reply:
It's redundant and it's the same except for `take(10)`